manifests/supervisorcluster/1.22/cns-csi.yaml

```diff
@@ -204,20 +204,19 @@ spec:
               topologyKey: "kubernetes.io/hostname"
       serviceAccount: vsphere-csi-controller
       nodeSelector:
-        node-role.kubernetes.io/master: ''
```

Comment:
why are we updating 1.22 yaml for k8s 1.24 change?

Reply:
+1